dicom_util/src/main/java/com/google/cloud/healthcare/IDicomWebClient.java

```diff
@@ -0,0 +1,28 @@
+package com.google.cloud.healthcare;
+
+import com.github.danieln.multipart.MultipartInput;
+import java.io.InputStream;
+import org.json.JSONArray;
+
+public interface IDicomWebClient {
```

Comment:
Just wondering, why was this changed to an interface? Do you plan to create a mock version in a future CL?

Reply:
Yes, c-find and c-move unit tests use this for mocks.